app/src/org/commcare/fragments/personalId/PersonalIdPhoneVerificationFragment.java

```diff
@@ -57,6 +62,7 @@ public class PersonalIdPhoneVerificationFragment extends BasePersonalIdFragment
     private PersonalIdSessionData personalIdSessionData;
     OtpVerificationCallback otpCallback;
     private ActivityResultLauncher<Intent> smsConsentLauncher;
+    private String lastOtpMethod = SMS_METHOD_FIREBASE;
```

Comment:
we should not be giving it a default value here as it doesn't really have a default, the initial method can be firebase or twilio/personalId based on server response.